packages/lexical-devtools/src/lexicalForExtension.ts

```diff
@@ -28,7 +28,9 @@ function getActiveEditorState(): lexical.EditorState {
       'Unable to find an active editor state. ' +
         'State helpers or node methods can only be used ' +
         'synchronously during the callback of ' +
-        'editor.update() or editorState.read().',
+        'editor.update() or editorState.read(). ' +
+        'This issue can also occur if you have a mismatching ' +
```

Comment:
IMO, this is more technically correct:

```suggestion
        'This issue can also occur if you have more than one copy of lexical in the same JS bundle ' +
        'or you're trying to update/read state of lexical that belongs to another JS bundle.'
```